### DISCUSSION TITLE
Filtering GET /collections request fails

### DISCUSSION THREAD

**Discussion by Tonton-Blax** [directus/directus#16937] [Feature Requests]
### Describe the Bug

Filter passed in query string does not work when querying the collection list ( `GET /collections` ).
All the collections are returned no matter what filter is applied.

### To Reproduce

Make a `GET` request on collections and apply a basic filter like:

`{URL}/collections?filter[collection][_contains]=posts`

This request works with any other data type, but not with `collections` per se.

### Errors Shown

The unfiltered payload is returned 
```
{
  data: [
    { collection: 'posts', meta: [Object], schema: [Object] },
    { collection: 'filiales', meta: [Object], schema: [Object] },
    { collection: 'formulaires', meta: [Object], schema: [Object] },
    { collection: 'languages', meta: [Object], schema: [Object] },
    { collection: 'directus_panels', meta: [Object], schema: [Object] },
...
    { collection: 'directus_shares', meta: [Object], schema: [Object] },
    { collection: 'directus_flows', meta: [Object], schema: [Object] }
  ]
}
```

### What version of Directus are you using?

9.21.2

### What version of Node.js are you using?

19.2.0

### What database are you using?

sqlite

### What browser are you using?

chrome

### How are you deploying Directus?

locally

**Comment by br41nslug** [comment]
Moving this to a feature request as it is documented that this does not work on the `/collections` endpoint.

> This endpoint doesn't currently support any query parameters.

https://docs.directus.io/reference/system/collections.html#query-parameters

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
